package.json

```diff
@@ -25,7 +25,7 @@
     "ws": "^7.4.2"
   },
   "peerDependencies": {
-    "zwave-js": "^6.5"
+    "zwave-js": "^6.6"
```

Comment:
We should do this in a different PR so the package-lock is also in that PR. In case we need to revert this PR, we don't want to revert the package upgrade.

Reply:
I can do that, but given that the schema change goes hand in hand with the `zwave-js` bump, wouldn't we want them to be combined? It's ok for the schema change to go out before the version bump, but not ok for the version bump to go out before the schema change

Reply:
oh good point.